pkgs/by-name/hd/hdrop/package.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  makeWrapper,
+  scdoc,
+  coreutils,
+  util-linux,
+  jq,
+  libnotify,
+  hyprland ? null,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "hdrop";
+  version = "0.2.3";
+
+  src = fetchFromGitHub {
+    owner = "Schweber";
+    repo = "hdrop";
+    rev = "9c29164";
+    sha256 = "sha256-aUL6mZ061rZBah8+/m5Guw23O7YeNK+FZY2obkVTk1I=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+    scdoc
+  ];
+
+  makeFlags = ["PREFIX=$(out)"];
+
+  postInstall = ''
+    wrapProgram $out/bin/hdrop --prefix PATH ':' \
+      "${lib.makeBinPath ([
+        coreutils
+        util-linux
+        jq
+        libnotify
+      ]
+      ++ lib.optional (hyprland != null) hyprland)}"
+  '';
+
+  meta = with lib; {
+    description = "Emulate 'tdrop' in Hyprland (run, show and hide specific programs per keybind)";
+    homepage = "https://github.com/Schweber/hdrop";
+    changelog = "https://github.com/Schweber/hdrop/releases/tag/${version}";
+    license = licenses.agpl3;
+    platforms = platforms.unix;
```

```suggestion
    platforms = platforms.linux;
```

Since some of your dependencies are Linux-only.